test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -21,10 +21,14 @@ public void elementsAreUnique() throws Exception {
         assertAreUnique(1);
         assertAreUnique("");
         assertAreUnique("abc");
+        assertAreUnique(2.0);
```

Comment:
The LO for this week is to write out test cases for the `Utils.isAnyNull` method, not the `assertAreUnique` method.

These assertions should be done for `Utils.isAnyNull` and should not be within the elementsAreUnique test.